include/mbedtls/ccm.h

```diff
@@ -1,10 +1,19 @@
 /**
  * \file ccm.h
  *
- * \brief Counter with CBC-MAC (CCM) for 128-bit block ciphers
+ * \brief AES CCM combines Counter mode encryption with CBC-MAC authentication 
+ * for 128-bit block ciphers.
```

Comment:
I think the comment shouldn't mention AES.

Reply:
fixed